client/src/main/java/org/spine3/util/Math.java

```diff
@@ -20,6 +20,8 @@
 
 package org.spine3.util;
 
+import static java.lang.String.*;
```

Comment:
Do not `static` use imports like this, specify a particular import. For example, use `java.lang.String.format`. Do it in all similar cases.